### PR TITLE
Improve messages viewmodel to retrieve results from api 

### PR DESCRIPTION
**Pull request by scdekov**
@kambata940 @SteinPowaaa

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -14,4 +14,8 @@
 //= require jquery_ujs
 //= require bootstrap-sprockets
 //= require turbolinks
-//= require_tree .
+//= require lib/knockout-3.4.0
+//= require lib/arbiter
+//= require bindings/scrollToBottom
+//= require bindings/toggler
+//= require view-models/chat
```
Comment (kanevk) [inline app/assets/javascripts/application.js:21]: Are you sure that we will need the chat in all pages, because `application.js` is load for all pages?

Comment (mihaildono) [inline app/assets/javascripts/application.js:21]: Pretty much all pages, except maybe a front page(pre-register)?

Comment (kanevk) [inline app/assets/javascripts/application.js:21]: So it is not needed everywhere and should be included only where it's needed.

File: app/assets/javascripts/bindings/scrollToBottom.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+ko.bindingHandlers.scrollToBottomOnChange = {
+    update: function(element, valueAccessor, allBindings, viewModel, bindingContext) {
+        ko.unwrap(valueAccessor());
+        element.scrollTop = element.scrollHeight;
+    }
+};
```
Comment (kanevk) [inline app/assets/javascripts/bindings/scrollToBottom.js:6]: Here should have a new line.

Comment (scdekov) [inline app/assets/javascripts/bindings/scrollToBottom.js:6]: Where? At the end of the file ?

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (scdekov) [inline app/assets/javascripts/application.js:21]: It is needed only on poker-room page, but we need to know the correct way to include the assets.

File: app/assets/javascripts/bindings/scrollToBottom.js (hunk 2 above)
Comment (kanevk) [inline app/assets/javascripts/bindings/scrollToBottom.js:6]: :+1:

**Comment by kanevk** [comment]
I merge it like this because it's not merge in master.

**Pull request merged**